x-pack/plugins/enterprise_search/public/applications/app_search/components/credentials/credentials_flyout/form_components/key_type.test.tsx

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { setMockValues, setMockActions } from '../../../../../__mocks__/kea.mock';
+
+import React from 'react';
+import { shallow } from 'enzyme';
+import { EuiSelect } from '@elastic/eui';
+
+import { ApiTokenTypes, TOKEN_TYPE_INFO } from '../../constants';
+import { FormKeyType } from './';
```

Comment:
I like it being the last import! so that the import and test block follow each other, like this:

```
import { FormKeyType } from './';

describe('FormKeyType', () => {
```

although like I've mentioned, it's definitely not a hard and fast rule or anything

Reply:
Ah, that's what it was, gotcha. Thanks.